react/src/components/Information.tsx

```diff
@@ -0,0 +1,325 @@
+  const { token } = theme.useToken();
+
+  const backendaiclient = useSuspendedBackendaiClient();
+
+  let { data: licenseInfo, isLoading: isLoadingLicenseInfo } = useQuery<{
+    valid: boolean;
+    type: string;
+    licensee: string;
+    key: string;
+    expiration: string;
+  }>(
+    "licenseInfo",
+    () => {
+      return backendaiclient?.enterprise.getLicense();
+    },
+    {
+      // for to render even this fail query failed
+      suspense: false,
+    }
+  );
+
+  if (!licenseInfo) {
+    licenseInfo = {
+      valid: false,
+      type: t("information.CannotRead"),
+      licensee: t("information.CannotRead"),
+      key: t("information.CannotRead"),
+      expiration: t("information.CannotRead"),
+    };
+  }
+
+  const columnSetting: DescriptionsProps["column"] = {
+    xxl: 4,
+    xl: 4,
+    lg: 2,
+    md: 1,
+    sm: 1,
+    xs: 1,
+  };
+
+  return (
+    <Flex
+      direction="column"
+      align="stretch"
+      style={{ margin: token.marginSM, gap: token.margin }}
+    >
+      <Row gutter={[token.margin, token.margin]}>
+        <Col xs={24} xxl={12}>
+          <Card
+            style={{
+              height: "100%",
+            }}
+          >
+            <Descriptions
+              title={t("information.Core")}
+              bordered
+              column={columnSetting}
+            >
+              <Descriptions.Item
+                label={
+                  <DescriptionLabel title={t("information.ManagerVersion")} />
+                }
+              >
+                <Flex
+                  direction="column"
+                  style={{ gap: token.marginXXS }}
+                  align="start"
+                >
+                  Backend.AI {backendaiclient.managerVersion}
+                  <DoubleTag
+                    label={t("information.Installation")}
+                    value={backendaiclient.managerVersion}
+                  />
+                  {/* TODO: get manager_version_latest  */}
+                  {/* <DoubleTag
+                label={t("information.LatestRelease")}
+                value={"manager_version_latest"}
+              /> */}
+                </Flex>
+              </Descriptions.Item>
+              <Descriptions.Item
+                label={<DescriptionLabel title={t("information.APIVersion")} />}
+              >
+                {backendaiclient.apiVersion}
+              </Descriptions.Item>
+            </Descriptions>
+          </Card>
+        </Col>
+        <Col xs={24} xxl={12}>
+          <Card>
+            <Descriptions
+              title={t("information.Security")}
+              bordered
+              column={columnSetting}
+            >
+              <Descriptions.Item
+                label={
+                  <DescriptionLabel
+                    title={t("information.DefaultAdministratorAccountChanged")}
+                    subtitle={t(
+                      "information.DescDefaultAdministratorAccountChanged"
+                    )}
```

Comment:
If there is only an icon, how about aligning the icon with the center?
```suggestion
                    )}
                    style={{ textAlign: "center" }}
```

Reply:
I think that aligning to the left would be better, since the column width is responsive. For now, keep the left align, change it later if necessary.